### ISSUE TITLE
Pentadactyl not available on Firefox 38.0

### ISSUE THREAD

**Issue by anujmaurice** [5digits/dactyl#53]
I get the following message
 "This add-on is not compatible with your version of Firefox because of the following: Add-on has opted into strict compatibility checking."

https://addons.mozilla.org/en-US/firefox/addon/pentadactyl/

**Comment by jgkamat** [comment]
I would suggest you try out the [nightly builds](http://5digits.org/nightlies), but the release version should be updated anyway

**Comment by willsALMANJ** [comment]
Yes, issue #26 raises these issues. For now, it is best to use the nightly build. There are some changes to Firefox that make version 1.1 not work well with Firefox 38.
